src/components/header/Header.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import './header.css'
+import {Link} from 'react-router-dom'
+
+export const Header = () => {
+  return (
+      <>
+        <div className='header'>
+            <Link to="/" className='brand-title'>
+            Neos- Quiz
+            </Link>
+
+            <Link to='/' className='guide-button'>
```

Comment:
Give correct path

Reply:
ya... as soon as  I will make the guide page , I will update it